app/ledger/n3NeonLedger.js

```diff
@@ -198,9 +209,10 @@ export const getStartInfo = async () => {
   const ledger = await NeonLedger3.init()
 
   try {
-    const deviceInfo = await ledger.getDeviceInfo()
-    const publicKey = await ledger.getPublicKey()
-    return { deviceInfo, publicKey }
+    return await Promise.all([
+      ledger.getDeviceInfo(),
+      ledger.getPublicKey(),
+    ]).then(([deviceInfo, publicKey]) => ({ deviceInfo, publicKey }))
```

Comment:
`const [deviceInfo, publicKey] = await Promise.all(...)`

Reply:
resolved in b49c341178a97650cf73ab078f3d2b95fb16e521